addon_service/authorized_storage_account/models.py

```diff
@@ -76,6 +76,12 @@ class AuthorizedStorageAccount(AddonsServiceBaseModel):
         related_name="authorized_storage_accounts",
     )
 
+    is_oauth1_ready = models.BooleanField(
```

Comment:
looks like we need this because we lack another way to distinguish between temporary and permanent credentials? would it be appropriate to add a field for tracking this to the OAuth1TokenCredentials?

Reply:
You are right, this is the purpose for this field. Your approach is also possible